test_runner/fixtures/compare_fixtures.py

```diff
@@ -333,3 +338,27 @@ def neon_with_baseline(request: FixtureRequest) -> PgCompare:
     fixture = request.getfixturevalue(request.param)
     assert isinstance(fixture, PgCompare), f"test error: fixture {fixture} is not PgCompare"
     return fixture
+
+
+@pytest.fixture(scope="function", autouse=True)
+def sync_after_each_test():
+    # some of the benchmarks are quite write-happy, and create issues to start the processes up in 10s.
+    # SYNC_AFTER_EACH_TEST is only "true" for benchmarks, which run test_runner/performance cases sequentially
```

Comment:
It seems the comment is slightly misleading, it's true for benchmarks only on CI.

Also, this fixture will be applied to all tests (even to regression): https://github.com/neondatabase/neon/blob/0205ce184967f4510b6034bf2051a495bf464b44/test_runner/conftest.py#L10

Reply:
I agree it will be enabled for all tests.

The env is used to enable it only for enabling in the sequential benchies as you surely saw.

Please suggest a better comment? :)

Reply:
I did do a small change, but I guess the problem you see still exists?

Reply:
Another attempt to fix the comment (and get another benchmark run) pushed.